docs/source/tracking.rst

```diff
@@ -265,12 +267,13 @@ The backend store is where MLflow Tracking Server stores experiment and run meta
 params, metrics, and tags for runs. MLflow supports two types of backend stores: *file store* and
 *database-backed store*.
 
-Use ``--backend-store-uri`` to configure type of backend store. This can be a local path *file
-store* specified as ``./path_to_store`` or ``file:/path_to_store``, or a SQL connection string
-for a *database-backed store*. For the latter, the argument must be a SQL connection string
-specified as ``db_type://<user_name>:<password>@<host>:<port>/<database_name>``. Supported
-database types are ``mysql``, ``mssql``, ``sqlite``, and ``postgresql``.
-For backwards compatibility, ``--file-store`` is an alias for ``--backend-store-uri``.
+Use ``--backend-store-uri`` to configure the type of backend store. A *file store* backend is
+specified as ``./path_to_store`` or ``file:/path_to_store``., while a *database-backed store* is
+specified with a `SQLAlchemy database URI <https://docs.sqlalchemy.org/en/latest/core/engines.html#database-urls>`_.
+The database URI typically takes the format ``<dialect>+<driver>://<username>:<password>@<host>:<port>/<database>``.
```

Comment:
A *file store* backend is
specified as ``./path_to_store`` or ``file:/path_to_store``., while a *database-backed store* is
specified with a `SQLAlchemy database URI ->

You specify a *file store* backend  as ``./path_to_store`` or ``file:/path_to_store`` and a *database-backed store* as a `SQLAlchemy database URI